tests/e2e/logging/test_openshift-logging.py

```diff
@@ -0,0 +1,115 @@
+
+    def finalizer():
+        teardown(sc_obj, cbp_obj)
+
+    request.addfinalizer(finalizer)
+
+    # Deploys elastic-search operator on the project openshift-operators-redhat
+    obj.create_namespace(yaml_file=constants.EO_NAMESPACE_YAML)
+    assert obj.create_elasticsearch_operator_group(
+        yaml_file=constants.EO_OG_YAML,
+        resource_name='openshift-operators-redhat'
+    )
+    assert obj.set_rbac(
+        yaml_file=constants.EO_RBAC_YAML, resource_name='prometheus-k8s'
+    )
+    assert obj.create_elasticsearch_subscription(constants.EO_SUB_YAML)
+
+    # Deploys cluster-logging operator on the project openshift-logging
+    obj.create_namespace(yaml_file=constants.CL_NAMESPACE_YAML)
+    assert obj.create_clusterlogging_operator_group(
+        yaml_file=constants.CL_OG_YAML
+    )
+    assert obj.create_clusterlogging_subscription(
+        yaml_file=constants.CL_SUB_YAML
+    )
+
+    # Creates instance for cluster-logging
+    cbp_obj = helpers.create_ceph_block_pool()
+    sc_obj = helpers.create_storage_class(
+        interface_type=constants.CEPHBLOCKPOOL,
+        interface_name=cbp_obj.name,
+        secret_name=constants.DEFAULT_SECRET,
+        reclaim_policy="Delete"
+    )
+    assert sc_obj, f"Failed to create storage class"
+    assert obj.create_instance_in_clusterlogging(sc_name=sc_obj.name)
+
+    # Check the health of the cluster-logging
+    assert obj.check_health_of_clusterlogging()
+
+
+def teardown(sc_obj, cbp_obj):
+    """
+    The teardown will uninstall the openshift-logging from the cluster
+    """
+    sc_obj.delete()
+    cbp_obj.delete()
+    uninstall_cluster_logging()
+
+
+@pytest.mark.usefixtures(
+    test_fixture.__name__
+)
+class TestLogging_in_EFK_stack(E2ETest):
+    """
+    The class contains the testcases related to openshift-logging
+    """
+    @pytest.mark.polarion_id("OCS-651")
+    @tier1
+    def test_respin_elasticsearch_pod(self):
+        """
+         The function respins the elasticsearch pod and try to get the
+         .operations index.
+
+        """
+        pod_list = get_all_pods(namespace='openshift-logging')
+        es_pod_list = [
+            pod.name for pod in pod_list if pod.name.startswith('elasticsearch')
+        ]
+
+        elasticsearch_pod_obj = get_pod_obj(
+            name=es_pod_list[1], namespace='openshift-logging'
+        )
+
+        elasticsearch_pod_obj.delete(force=True)
+
+        es_pod_obj = get_pod_obj(
+            name=es_pod_list[2], namespace='openshift-logging'
+        )
+        operations_index = es_pod_obj.exec_cmd_on_pod(
+            command='es_util --query=.operations.*/_search?pretty', out_yaml_format=True
+        )
+        logger.info(operations_index)
+        pod_list = get_all_pods(namespace='openshift-logging')
+        es_pod_list_after_respin = [
+            pod.name for pod in pod_list if pod.name.startswith('elasticsearch')
+        ]
+        if len(es_pod_list_after_respin) == len(es_pod_list):
```

Comment:
The validation done here is whether es pod exists. There is no validation of whether es pod is running. Please add the following validation. 
1) validate all es pods are running after respin
2) validate es cluster is healthy (green)
3) New logs are stored
This test is invalid without these checks.

Reply:
Done

Reply:
done